src/index.js

```diff
@@ -26,6 +26,18 @@ const sleep = (ms) => new Promise(resolve => setTimeout(resolve, ms));
 
 export async function run() {
   try {
+    // custom working directory
+    const workingDirectoryInput = core.getInput('working-directory');
```

Comment:
This new `getInput()` call is unexpected by the tests and therefore causes [this test failure](https://github.com/mxschmitt/action-tmate/pull/157/checks#step:5:129). Please adjust the tests to expect that call.